css/style.css

```diff
@@ -1,10 +1,142 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
-body {
+ body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
 }
+header {
+  position: fixed;
+  top: 0;
+  min-height: 75px;
+  padding: 0px 50px;
+  display: flex;
+  justify-content: flex;
+  align-items: center;
+  background-color: #ffffff;
+ 
+}
+@media (max-width: 500px) {
+  header {
+    flex-wrap: wrap;
+  }
+}
+
+  
+.logo {
+    margin-top: 15px;
+    width: 10%;
+    position: relative;
+  }
+
+
+.logo > img {
+  width: 100%;
+  height: 3.375rem;
+  max-width: 200px;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
```

Comment:
These rules only work when applied to a parent element, which wraps around child elements.
The image is not a parent so these rules have no effect.

Remember, when you want to move elements with flexbox, apply the flexbox rules to the **parent** element, not the element you want to move!